### PR TITLE
net/chrony: add diagnostics

### LINKED ISSUE #2209: net/chrony: Add diagnostics
Note to self:

Add Diagnostics to chrony plugin as it seems to get more popular:

https://forum.opnsense.org/index.php?topic=21095.msg98665#msg98665

### PR DESCRIPTION
**Pull request by mimugmail**
Closes #2209

### CODE REVIEW

**Comment by mimugmail** [comment]
image

File: net/chrony/src/opnsense/mvc/app/controllers/OPNsense/Chrony/Api/ServiceController.php [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?php
 
 /*
- * Copyright (C) 2020 Michael Muenz <m.muenz@gmail.com>
+ * Copyright (C) 2020 - 2021 Michael Muenz <m.muenz@gmail.com>
```
Comment (fichtner) [inline net/chrony/src/opnsense/mvc/app/controllers/OPNsense/Chrony/Api/ServiceController.php:4]:
```suggestion
 * Copyright (C) 2020-2021 Michael Muenz <m.muenz@gmail.com>
```

File: net/chrony/src/opnsense/mvc/app/views/OPNsense/Chrony/general.volt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {#
  # Copyright (c) 2019 Deciso B.V.
- # Copyright (c) 2020 Michael Muenz <m.muenz@gmail.com>
+ # Copyright (c) 2020 - 2021 Michael Muenz <m.muenz@gmail.com>
```
Comment (fichtner) [inline net/chrony/src/opnsense/mvc/app/views/OPNsense/Chrony/general.volt:3]:
```suggestion
 # Copyright (c) 2020-2021 Michael Muenz <m.muenz@gmail.com>
```

File: net/chrony/src/opnsense/service/conf/actions.d/actions_chrony.conf [hunk 3]
```diff
@@ -21,3 +21,27 @@ command:/usr/local/etc/rc.d/chronyd status;exit 0
 parameters:
 type:script_output
 message:request chrony status
+
+[chronysources]
```
Comment (fichtner) [inline net/chrony/src/opnsense/service/conf/actions.d/actions_chrony.conf:25]: I’m unsure we should use chrony prefix here, configctl chrony chronysources looks weird. But not a blocker.

**Review by fichtner — APPROVED** [review]

**Comment by fichtner** [comment]
Merged, thanks!

**Pull request merged**